features/step_definitions/networking.rb

```diff
@@ -783,7 +783,7 @@
 
   step "the default interface on nodes is stored in the clipboard"
   step "I run command on the node's sdn pod:", table(
-    "| bash | -c | ip a show \"<%= cb.interface %>\" \\| grep inet \\| grep -v inet6  \\| awk '{print $2}' |"
+    "| bash | -c | ip a show \"<%= cb.interface %>\" \\| grep inet \\| grep -v inet6  \\| awk '{print $2}' \\| head -1 |"
```

Comment:
You can use `-4` to only show IPv4 address, and `-brief` removes all the extra junk

```
$ oc exec sdn-8cgws -- ip -4 -brief a show ens5
ens5             UP             10.0.180.62/19 192.168.1.44/24
```

```
$ oc exec sdn-8cgws -- ip -4 -brief  a show ens5 | awk '{print $3}'
10.0.180.62/19
```

`"| bash | -c | ip -4 -brief a show \"<%= cb.interface %>\"  \\| awk '{print $3}' |"`

Reply:
Thanks @rbbratta !